front/components/sparkle/AppLayout.tsx

```diff
@@ -172,6 +216,30 @@ export const SidebarProvider = ({
   );
 };
 
+export const NavigationBarContext = React.createContext<{
```

Comment:
Why do we need a Provider? that seems very local to AppLayout no?

Reply:
Yes that's why I was wondering if having a context here was relevant or not

Reply:
If you don't need it, don't use it :)

Reply:
Indeed, a simple state works 😸